cla_backend/libs/eligibility_calculator/calculator.py

```diff
@@ -345,26 +345,54 @@ def is_disposable_capital_eligible(self):
         return self.disposable_capital_assets <= limit
 
     def is_eligible(self):
-        self._do_cfe_civil_check()
-
-        legacy_result = self._legacy_check()
-        logger.info("Eligibility result (legacy): %s %s" % (legacy_result, self.calcs))
+        cfe_result, cfe_details = self._do_cfe_civil_check()
+
+        # temp: start doing just non-partner passported checks through CFE
+        if self.case_data.facts.on_passported_benefits and hasattr(self.case_data.facts,
+                                                                   "has_partner") and not self.case_data.facts.has_partner:
+            self.calcs = {
+                "pensioner_disregard": int(cfe_details.pensioner_disregard * 100),
+                "disposable_capital_assets": int(cfe_details.disposable_capital_assets * 100),
+                "property_equities": [int(x * 100) for x in cfe_details.property_equities],
+                "property_capital": int(cfe_details.property_capital * 100),
+                "liquid_capital": int(cfe_details.liquid_capital * 100),
+                "gross_income": 0,
+                "partner_allowance": 0,
+                "disposable_income": 0,
+                "dependants_allowance": 0,
+                "employment_allowance": 0,
+                "partner_employment_allowance": 0,
+            }
+
+            return cfe_result
```

Comment:
Rather than return here, can we both checks, so we can look at the logs during use and (visually at least) compare the outputs of CFE and legacy_check()?